src/main/java/roomescape/dao/TimeDao.java

```diff
@@ -0,0 +1,55 @@
+package roomescape.dao;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Time;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class TimeDao {
+    private final JdbcTemplate jdbcTemplate;
+
+    public TimeDao(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    private final RowMapper<Time> timeRowMapper = (resultSet, rowNum) ->
+            new Time(
+                    resultSet.getLong("id"),
+                    resultSet.getString("time")
+            );
+
+    public Time createTime(String time) {
+        String sql = "INSERT INTO time (time) VALUES (?)";
+
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(sql, new String[]{"id"});
+            ps.setString(1, time);
+            return ps;
+        }, keyHolder);
+
+        Long id = keyHolder.getKey().longValue();
+        return new Time(id, time);
+    }
+
+    public List<Time> findAllTimes() {
+        String sql = "SELECT id, time FROM time";
+        return jdbcTemplate.query(sql, timeRowMapper);
+    }
+
+    public Time findById(Long id) {
+        String sql = "SELECT id, time FROM time WHERE id = ?";
+        return jdbcTemplate.queryForObject(sql, timeRowMapper, id);
+    }
+
+    public void deleteId(Long id) {
+        String sql = "DELETE FROM time WHERE id = ?";
+        jdbcTemplate.update(sql, id);
+    }
```

Comment:
```suggestion
    public Time findById(Long id) {
        String sql = "SELECT id, time FROM time WHERE id = ?";
        return jdbcTemplate.queryForObject(sql, timeRowMapper, id);
    }

    public void deleteById(Long id) {
        String sql = "DELETE FROM time WHERE id = ?";
        jdbcTemplate.update(sql, id);
    }
```
- 통일성을 위해 바꾸면 더 좋을 것 같아요!

Reply:
음 메서드명을 By를 빼고 적었네요..꼼꼼한 확인 감사합니다